places/models.py

```diff
@@ -7,3 +9,20 @@ class Artist(models.Model):
 
     def __str__(self):
         return self.name
+
+class Work(models.Model):
+    title = models.CharField(max_length=200, blank=True, null=True)
+    creators = models.ManyToManyField(Artist, related_name="works")
+    year = models.IntegerField(blank=True, null=True)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True)
+    category = models.CharField(max_length=20, blank=True, null=True)
+    description = models.TextField(blank=True, null=True)
+    # artistStatement = soundbyte?
+    photo = models.ImageField
+    artistWebsite = models.URLField(blank=True, null=True)
```

Comment:
Artist website is not needed  because the field is under the Artist class.

Reply:
Great point, fixed it!